proj.py

```diff
@@ -0,0 +1,123 @@
+import argparse
+import matplotlib.pyplot as plt
+import numpy as np
+import os
+import pandas as pd
+import seaborn as sns
+import sqlite3
+
+
+def load_data():
+    """loads the required .csv files."""
+    return pd.read_csv(r'C:\Users\Dell\OneDrive\Desktop\netflix_titles.csv')
+
+
+def db_connection():
+    """creates a connection to the database."""
+    return sqlite3.connect("netflix.db")
+
+
+def init_app():
+    """initializes all the required variables"""
+    temp_df = load_data()
```

Comment:
don't use the temp keyword this looks ugly in code. you simply use `df` in multiple functions because the scope of `df` would remain within this function only.

Reply:
temp keywords removed, was avoiding usage of df alone as it was shadowing the variable in outer scope but I understand that it wouldn't cause any issues outside of the function as its scope ends with the function. Duly noted!